conf/messages

```diff
@@ -33,9 +33,6 @@ global.webchat.unavailable=Webchat is not available at the moment.
 global.banner.phase = BETA
 
 #Webchat pages
```

Comment:
there's nothing in this section it seems. Can we remove this?

Reply:
Done